Automattic/jetpack
# Verbum: Update Verbum comments accessibility

Fixes p4vt7e-io-p2

## Proposed changes:

* Adds visible focus to WordPress.com, Facebook, and Email buttons
* Fixes email input fields visual focus (certain themes were causing a break in the UI)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync to simple site and atomic site
* Verify Verbum Comment has accessibility visual indicators ( Logout, "X" close button, login icons, email input fields, settings toggle, comment button)
